week-1/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,6 @@
 // Complete the function so that it takes input parameters
 function multiply() {
-  // Calculate the result of the function and return it
+  return 3 * 4;
```

Comment:
Here, we want to use the values that are passed into the function. So you need to modify it from ` function multiply() {` to ` function multiply(num1, num2) {` and change the return statement too. I hope this makes sense.